### ISSUE TITLE
Not Detecting Removed by Moderator Posts in Old Reddit

### ISSUE THREAD

**Issue by knakamura8** [DenverCoder1/Unedit-for-Reddit#2]
Script is not detecting removed by moderator posts in old reddit.

Example:  https://old.reddit.com/r/answers/comments/pwr9cy/what_happened_to_removedditcom_and_cedditcom/hela727/

Verification that the data for these comments remains extant on pushshift:  https://www.reveddit.com/v/answers/comments/pwr9cy/what_happened_to_removedditcom_and_cedditcom/hela727/?ps_after=1637818060

There is a check for `x.title.substr(0,11) == "last edited";`, yet not one for [deleted] and/or [removed].

By the way, thank you for the script :)

**Comment by jamesWalker55** [comment]
Here's a quick and buggy fix, it indiscriminately adds the "Show original" button to all posts, no matter they are deleted or not. Replace the else block at line 267 with:

```javascript
        } else {
        /* Old Reddit */
            /* edited comments and submissions */
            editedComments = Array.from(document.querySelectorAll("time")).filter(function (x, y, z) {
                const notFound = Array.from(x.classList).indexOf("found") < 0;
                const isEdited = x.title.substr(0, 11) == "last edited";
                if (notFound && isEdited) return isEdited;
                
                const isPost = x.parentElement.parentElement.classList.contains("entry");
                return notFound && isPost;
            });
        }
```

**Comment by knakamura8** [comment]
Thank you!

**Comment by DenverCoder1** [comment]
Should be fixed now by #4
